app/wp-content/themes/kstg/library/js/vehicle.js

```diff
@@ -41,7 +53,34 @@ var calloutValue = $('.specialFeature').text();
         $('.next').prop('disabled', false);
       }
     }
+    function setEventListeners() {
+      $('.next').on('click', function() {
```

Comment:
Maybe not something that you need to do right away, but consider doing something like you did for the image click handler and move these into functions that get called above. Example:

`$('.next').on('click', advanceCarousel)`

You don't have to, but it would just keep your `setEventListeners` function clean so that wen someone looks at it, they can immediately get an idea as to what you're adding listeners to (the point of the function).